docs/databases/connections/mongodb.md

```diff
@@ -51,13 +51,54 @@ Metabase currently does NOT support the following connection string parameters:
 - **Rerun queries for simple exploration**: When this is on, Metabase will automatically run queries when users do simple explorations with the Summarize and Filter buttons when viewing a table or chart. You can turn this off if querying this database is slow. This setting doesn’t affect drill-throughs or SQL queries.
 - **Choose when syncs and scans happen**: By default, Metabase does a lightweight hourly sync and an intensive daily scan of field values. If you have a large database, we recommend turning this on and reviewing when and how often the field value scans happen.
 - **Periodically refingerprint tables**: This setting — disabled by default — enables Metabase to scan for additional field values during syncs allowing smarter behavior, like improved auto-binning on your bar charts.
 
+## Connecting to a MongoDB Atlas cluster
+
+> You can use Metabase with Atlas Dedicated and Shared clusters. Metabase does not currently support Atlas Serverless.
+
+### Whitelist IP addresses
+
+If you are using Metabase Cloud, you'll need to whitelist [Metabase Cloud IP addresses](https://www.metabase.com/docs/latest/cloud/ip-addresses-to-whitelist) in your Atlas cluster. If you are using self-hosted Metabase, you'll need to whitelist the IP of your Metabase instance.
+
+1. Log into your [Atlas cluster](https://cloud.mongodb.com)
+2. Go to **Network Access**
+3. Add the IP addresses that your Metabase uses to connect.
+
+### Connect Metabase to your cluster
+
+> The connection string provided in Atlas "Connect" interface does not include the database. Metabase requires you to provide a database name when connecting, so you'll need to edit the connection string to add the database name.
+
+1. Log into your [Atlas account](https://cloud.mongodb.com)
+
+2. Select the cluster you want to connect to, and click **Connect**.
+
+   ![Your cluster screengrab](../images/atlas-connect.png)
+
+3. Select **Drivers**.
+
+4. Copy the connection string from **Add your connection string into your application code** section
+
+   ![Connect screengrab](../images/connection-string.png)
+
+5. In Metabase, go to Admin -> Databases, and click the **Add database** button.
+6. Select MongoDB from the dropdown, and enter your desired **Display name** for this database.
+7. Click on **"Paste the connection string"** and paste your connection string.
+8. Edit the connection string to include the name of the database after `/`:
+
+   ```
+   mongodb+srv://metabot:metapass@my-test-cluster.a5ej7.mongodb.net/DATABASE_NAME?retryWrites=true&w=majority&appName=my-test-cluster
+   ```
+
+See more information about [Advanced options](#settings-common-to-both-connection-options).
+
+If you are using Metabase fields to input connection information for your Atlas cluster instead of using the connection string, you might need to turn on **Use DNS SRV when connecting**.
```

Comment:
```suggestion
If you're using Metabase fields to input connection information for your Atlas cluster instead of using the connection string, you might need to turn on **Use DNS SRV when connecting**.

See more information about [Advanced options](#settings-common-to-both-connection-options).
```